packages/demand-bloc-payment/src/providers/AdyenProvider.ts

```diff
@@ -10,6 +10,7 @@ import {
   CompleteThreeDSecureVerificationParams,
   Payer,
   AdyenShopperData,
+  AUTHORISED,
```

Comment:
could we do something to get ResultCodes.AUTHORISED in a condition ? Just AUTHORISED is not very clear